### ISSUE TITLE
Example dataset and fine-tuned model for reproduction?

### ISSUE THREAD

**Issue by gameveloster** [mzbac/qlora-fine-tune#5]
Do you have an example dataset, the training arguments, the training loss curve and/or the fine-tuned model?

I am trying to try to reproduce a good qlora model on my system, if you can provide an example it will be very beneficial. Thanks!
